.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is marked for ignoring, but ensure that this is intentional. If your project requires media files to be versioned, you might want to remove this line. Otherwise, if media files are generated or should not be tracked, this is correct.